sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/transform/BeamBuiltinAggregations.java

```diff
@@ -397,6 +399,93 @@ public Long extractOutput(Long accum) {
    * <p>Note: null values are ignored when mixed with non-null values.
    * (https://issues.apache.org/jira/browse/BEAM-10379)
    */
+  //  static class BitAnd<T extends Number> extends CombineFn<T, Long, Long> {
```

Comment:
I think this commented-out implementation of BitAnd was left over from a stale commit. Please remove lines 402-488.